### PR TITLE
Add linter for reference files

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

File: .github/workflows/check_reference.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Check reference strings
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/check_reference_strings.yaml'
```
Comment (bcolsson) [inline .github/workflows/check_reference.yaml:7]: Inconsistent with this file name. Recommend updating the filename to `check_reference_strings.yaml` to match.

File: .github/workflows/check_reference.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      - '.github/scripts/linter_config.json'
+      - 'en-US/*.xliff'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/check_reference_strings.yaml'
```
Comment (bcolsson) [inline .github/workflows/check_reference.yaml:14]: Same as above. Recommend updating the filename to `check_reference_strings.yaml` to match.

**Review by bcolsson — COMMENTED** [review]
Looks good, just a discrepancy between the yaml filename which should be adjusted. (Unless I'm misunderstanding something.)

File: .github/workflows/check_reference.yaml (hunk 1 above)
Comment (flodolo) [reply .github/workflows/check_reference.yaml:7]: Good catch. I renamed the file because it felt too long, but forgot to update the yaml file.

Comment (flodolo) [reply .github/workflows/check_reference.yaml:7]: (I will also switch to `.yml`, since the other files in the repo are using that extension.)

**Pull request merged**